mappings/net/minecraft/network/encryption/ArgumentSignatures.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/class_7450 net/minecraft/network/encryption/ArgumentSignatures
```

Comment:
Just curious, what signature is this for? Seems messages and command arguments have their own signatures already.

Reply:
iirc, CAS.Signature is for individual arguments while this one is a collection of them, typical record name mojank

Reply:
Having nonobfscated fields visible helps, I remember seeing `Map<String, byte[]>`